### DISCUSSION TITLE
Notify not working?

### DISCUSSION THREAD

**Discussion by Stamperx** [rizwansoaib/whatsapp-monitor#157] [General]
Not sure if I’m doing it wrong but my notify run for WhatsApp monitor on Firefox is not working as it should . If I’m in contact with a person( ie I click on the chat ) and if the person is online I get a notification. But the contact I select doesn’t get me a notification otherwise even though he is online. WhatsApp web is left on in the background at all time. Any suggestions or help? Thanks

**Comment by rizwansoaib** [comment]
Please generate new Channel(key) and paste in extension. 

Maybe it will work.

**Reply by Stamperx** [reply]
Tried that but still not working as it used to. Not sure if WhatsApp web isn’t connecting well or it’s WhatsApp too. If it’s someone I’ve been having conversation often it picks up better. Have tried logging out and in too. If you got some time give it a test. Thanks for the great work!
